indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -233,6 +245,12 @@ public TaskStatus call()
                               command.add(String.format("-Ddruid.host=%s", childHost));
                               command.add(String.format("-Ddruid.port=%d", childPort));
 
+                              if(config.isSeparateIngestionEndPoint()) {
+                                command.add(String.format("-Ddruid.indexer.task.chathandler.service=%s", "druid/chathandler"));
```

Comment:
according to http://zookeeper.apache.org/doc/r3.1.2/zookeeperProgrammers.html `placeholder/../servicename` would be an invalid path.